tasks/us/census/lodes.py

```diff
@@ -561,11 +561,6 @@ def columns(self):
                 targets={total_jobs: 'denominator'},
                 tags=[tags['employment'], tags['commerce_economy']]
             )),
-            ('createdate', OBSColumn(
```

Comment:
Probably we have to make a bump in the version field because we're changing columns

Reply:
Version bumped

Reply:
Shouldn't be the version bump in the lodes columns?

Reply:
I have bumped both the `ColumnsTask` and the `TableTask` version

Reply:
I see the bump changes in the `spielman_singleton_segments.py` file :\

Reply:
Oops... solved